statistics/handle/handle.go

```diff
@@ -221,6 +221,29 @@ func (h *Handle) Update(is infoschema.InfoSchema) error {
 	return nil
 }
 
+// UpdateSessionVar updates the necessary session variables for the stats reader.
+func (h *Handle) UpdateSessionVar() error {
+	reader, err := h.getStatsReader(nil)
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		err1 := h.releaseStatsReader(reader)
+		if err1 != nil && err == nil {
+			err = err1
+		}
+	}()
+
+	verInString, err := reader.ctx.GetSessionVars().GlobalVarsAccessor.GetGlobalSysVar(variable.TiDBAnalyzeVersion)
```

Comment:
Can we use `h.mu.ctx.GetSessionVars()` directly?

Reply:
Oh, previously i want to use the lock operation in `get` and `release` reader to avoid data race.
Now i changed a way to impl.